rest-api/src/main/java/com/cloudbees/workflow/flownode/FlowNodeUtil.java

```diff
@@ -73,9 +65,7 @@ private FlowNodeUtil() {
     }
 
     public abstract static class CacheExtensionPoint implements ExtensionPoint {
-        public abstract Cache<String,List<FlowNode>> getExecutionCache();
```

Comment:
🐛 Changing of extension point APIs definitely requires 2.0 version of the plugin